.github/workflows/met-api-cd.yml

```diff
@@ -22,14 +21,13 @@ defaults:
 env:
   APP_NAME: "met-api"
   TAG_NAME: "${{ github.event.inputs.environment || 'dev' }}" # If the environment type is manually selected, use the input value; otherwise, use 'dev' as default
-  
+
   OPENSHIFT_LOGIN_REGISTRY: ${{ secrets.OPENSHIFT_LOGIN_REGISTRY }}
```

Comment:
Are these Openshift environment variables all new in this location? How were we accessing them before?